### PR TITLE
[Shop] Allow to choose province on addressing step

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | 
| License         | MIT

To-do:
- [ ] think about proper route and url for provinces field loading

### CODE REVIEW

File: src/Sylius/Behat/Page/Shop/Checkout/AddressingPage.php [hunk 1]
```diff
@@ -36,6 +37,13 @@ public function getRouteName()
      */
     public function chooseDifferentBillingAddress()
     {
+        $driver = $this->getDriver();
+        if ($driver instanceof Selenium2Driver) {
+            $this->getElement('different_billing_address_label')->click();
+
+q            return;
```
Comment (pamil) [inline src/Sylius/Behat/Page/Shop/Checkout/AddressingPage.php]: q XDD

Comment (Zales0123) [inline src/Sylius/Behat/Page/Shop/Checkout/AddressingPage.php]: q XDD 💃

Comment (pamil) [inline src/Sylius/Behat/Page/Shop/Checkout/AddressingPage.php]: q XDD 💃 👯

File: features/checkout/addressing_order/choosing_province_for_country_with_provinces.feature [hunk 2]
```diff
@@ -0,0 +1,43 @@
+@checkout
+Feature: Choosing province for country with provinces
```
Comment (tuka217) [inline features/checkout/addressing_order/choosing_province_for_country_with_provinces.feature]: Isn't it obvious that country has provinces if you can choose one?

**Comment by pjedrzejewski** [comment]
Thank you Mateusz!

**Pull request merged**